### PR TITLE
[15.0][ADD] stock_landed_costs_priority

### PR DESCRIPTION
**Pull request by LauraCForgeFlow**
This module adds priority to landed costs. Priorities can be used to rank landed costs by their level of importance, urgency or other criteria.

### CODE REVIEW

File: stock_landed_costs_priority/models/stock_landed_cost.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    _inherit = "stock.landed.cost"
+    _order = "priority desc, id"
+
+    priority = fields.Selection(
+        selection=[
+            ("0", "Normal"),
+            ("1", "Important"),
+        ],
+        default="0",
+        string="Important",
```
Comment (MiquelRForgeFlow) [inline stock_landed_costs_priority/models/stock_landed_cost.py]:
```suggestion
        string="Priority",
```

Comment (LauraCForgeFlow) [reply stock_landed_costs_priority/models/stock_landed_cost.py]: Fixed! :)

File: stock_landed_costs_priority/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+"PO-Revision-Date: 2024-11-04 13:43+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_landed_costs_priority
+#: model:ir.model.fields,field_description:stock_landed_costs_priority.field_stock_landed_cost__priority
```
Comment (MiquelRForgeFlow) [inline stock_landed_costs_priority/i18n/es.po]: remove this line

File: stock_landed_costs_priority/i18n/stock_landed_costs_priority.pot [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#. module: stock_landed_costs_priority
+#: model:ir.model.fields,field_description:stock_landed_costs_priority.field_stock_landed_cost__priority
+#: model_terms:ir.ui.view,arch_db:stock_landed_costs_priority.view_stock_landed_cost_search
+msgid "Priority"
+msgstr ""
+
+#. module: stock_landed_costs_priority
+#: model:ir.model,name:stock_landed_costs_priority.model_stock_landed_cost
+msgid "Stock Landed Cost"
+msgstr ""
```
Comment (MiquelRForgeFlow) [inline stock_landed_costs_priority/i18n/stock_landed_costs_priority.pot]: add blank line

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by JoanSForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**